hedera-node/hedera-app/src/main/java/com/hedera/node/app/OrderedServiceMigrator.java

```diff
@@ -40,7 +40,7 @@
  * The entire purpose of this class is to ensure that inter-service dependencies are respected between
  * migrations. The only required dependency right now is the {@link EntityIdService}, which is needed
  * for genesis blocklist accounts in the token service genesis migration. (See {@link
- * Service#registerSchemas(SchemaRegistry, SemanticVersion)}).
+ * RpcService#registerSchemas(SchemaRegistry, SemanticVersion)}).
```

Comment:
`RpcService`, while an inheriting interface of `Service`, doesn't directly contain the `registerSchemas(...)` method. I think we want this to stay as `Service` (instead of `RpcService`)

Reply:
Agreed